Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocControlObject.cs

```diff
@@ -92,17 +92,15 @@ protected IReadOnlyList<string> GetValidationErrors ([NotNull] ElementScope scop
       {
         return validationErrorsScope.InnerHTML.Split (new[] { validationErrorDelimiter }, StringSplitOptions.RemoveEmptyEntries);
       }
-      catch (MissingHtmlException exception)
+      catch (MissingHtmlException)
       {
-        throw CreateMissingValidationErrorFieldException (exception);
+        throw CreateMissingValidationErrorFieldException();

```

Comment:
locator

Reply:
Changed exception message to the following instead of providing a locator.
```csharp
MissingHtmlException ($"Could not find validation error field of the control object with the ID '{GetHtmlID()}'. This could be due to wrong markup or a missing validator.");
```
Note that GetHtmlID() could possibly return an empty string. I thought about providing the type if the control object instead, if the ID is empty. Should I implement it that way, or leave it as it is until starting [RM-7254](https://re-motion.atlassian.net/browse/RM-7254)?